README.md

```diff
@@ -10,7 +10,7 @@ SPDX-License-Identifier: MPL-2.0
 [![Downloads](https://static.pepy.tech/badge/power-grid-model)](https://pepy.tech/project/power-grid-model)
 [![Downloads](https://static.pepy.tech/badge/power-grid-model/month)](https://pepy.tech/project/power-grid-model)
 
-[![Build and Test C++ and Python](https://github.com/PowerGridModel/power-grid-model/actions/workflows/main.yml/badge.svg)](https://github.com/PowerGridModel/power-grid-model/actions/workflows/main.yml)
+[![Build and Test C++ and Python](https://github.com/PowerGridModel/power-grid-model/actions/workflows/build-test-release.yml/badge.svg)](https://github.com/PowerGridModel/power-grid-model/actions/workflows/build-test-release.yml)
```

Comment:
the nightly badge also needs to be updated

Reply:
I updated the links to ci.yml, but I left the name as the "Nightly build" since ci is still doing the nightly build

Reply:
i think you missed this
```suggestion
[![Build and Test C++ and Python](https://github.com/PowerGridModel/power-grid-model/actions/workflows/build-test-release.yml/badge.svg)](https://github.com/PowerGridModel/power-grid-model/actions/workflows/ci.yml)
```

Reply:
please do the same for the other badges. IF the original badges work (and i don't know whether it will), we can always revert to the original badges, but at least it should keep working

Reply:
also, the badge itself should also be fixed
```suggestion
[![Build and Test C++ and Python](https://github.com/PowerGridModel/power-grid-model/actions/workflows/ci.yml/badge.svg)](https://github.com/PowerGridModel/power-grid-model/actions/workflows/build-test-release.yml)
```